cmd/snap-bootstrap/cmd_initramfs_mounts.go

```diff
@@ -64,6 +65,25 @@ var (
 	osutilIsMounted = osutil.IsMounted
 )
 
+// XXX: make this more flexible if there are multiple seeds on disk, i.e.
+// read boot environment in this case
+func findSeed() (seedDir, seedLabel string, err error) {
```

Comment:
maybe better to s/seed/system/  or s/seed/recoverySystem/ here,  otherwise is a bit ambiguous with the /ubuntu-seed directory itself